portlets/src/main/webapp/vue-app/engagement-center/components/rules/RuleDetailDrawer.vue

```diff
@@ -267,6 +266,7 @@ export default {
       };
 
       this.$refs.ruleDetailDrawer.startLoading();
+      this.canAnnounce = true;
```

Comment:
No need to drawer loading to change here. In fact, it should be made inside a watch this way : 
```
watch: {
  sending() {
    if (this.sending) {
      this.$refs.ruleDetailDrawer.startLoading();
    } else {
      this.$refs.ruleDetailDrawer.endLoading();
    }
  }
}
```

```suggestion
      this.canAnnounce = true;
```

Reply:
done